docs/rc.xml.all

```diff
@@ -491,7 +501,7 @@
         <action name="ShowMenu" menu="root-menu" />
       </mousebind>
       <mousebind button="Middle" action="Press">
-        <action name="ShowMenu" menu="root-menu" />
+        <action name="ShowMenu" menu="client-list-combined-menu" />
```

Comment:
`rc.xml.all` reflect the built-in default keybinds/mousebinds, so suggest removing this line or adding is (the `client-list-combined-menu`) as an example.

Reply:
Done, I put back root-menu and made client-list-combined-menu a commented out example.